models/game.rb

```diff
@@ -83,7 +84,23 @@ def self.home_games(user, **opts)
     }.transform_values { |v| v&.to_i || 0 }
 
     opts[:user_id] = user.id if user
-    fetch(user ? LOGGED_IN_QUERY : LOGGED_OUT_QUERY, **opts,).all.sort_by(&:id).reverse
+    results = fetch(user ? LOGGED_IN_QUERY : LOGGED_OUT_QUERY, **opts,).all.sort_by(&:id).reverse
```

Comment:
i'm not sure why user context is necessary at all, is it just to show if a user is in a game?

don't you already know this?